golang/cosmos/go.mod

```diff
@@ -187,7 +187,7 @@ replace (
 // Agoric-specific replacements:
 replace (
 	// We need a fork of cosmos-sdk until all of the differences are merged.
-	github.com/cosmos/cosmos-sdk => github.com/agoric-labs/cosmos-sdk v0.46.16-alpha.agoric.2
+	github.com/cosmos/cosmos-sdk => github.com/agoric-labs/cosmos-sdk v0.46.16-alpha.agoric.2.1
```

Comment:
`.3` was already taken by future incompatible changes (waiting in the wings of an agoric-sdk PR, but I'm not sure which one).  `.2.1` implies the first published revision of `.2`, which is more accurate.

Reply:
We can't use the current tip of `Agoric` since lien support was removed in agoric-labs/cosmos-sdk#385 (in accordance with #8988, which is on `master` but not cherry-picked into the upgrade-14 release branch). So we need to use a side branch of `Agoric` for the new fee decorator feature. Our naming convention is to use `<cosmos-release>-alpha.agoric.N` sequentially on `Agoric`, `...-alpha.agoric.N.M` for sub-branches, and presumably `...N.M.P` for sub-sub-branches, etc. I'll document the policy in that repo.